CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.15.1 (2024-05-28)
+
+This release contains performance improvements and bug fixes since
+the 2.15.0 release. We recommend that you upgrade at the next
+available opportunity.
```

Comment:
Should probably wait for any wording changes by @billy-the-fish

Reply:
right.